src/stan/io/dump.hpp

```diff
@@ -269,8 +268,8 @@ class dump_reader {
   int get_int() {
     int n = 0;
     try {
-      n = boost::lexical_cast<int>(buf_);
-    } catch (const boost::bad_lexical_cast& exc) {
+      n = std::stol(buf_);
```

Comment:
```suggestion
      n = std::stoi(buf_);
```
?

imo can we back out these changes for `lexical_cast` for another PR?

Reply:
done